.flake8

```diff
@@ -6,3 +6,4 @@ extend-ignore =
     # Formatting style for `black`
     E203  # Whitespace before ':'
     W503  # Line break occurred before a binary operator
+    B023  # Function definition does not bind loop variable
```

Comment:
The ⛔ is because GitHub expects text files to end in one and only one `\n`.

Reply:
I don't understand

Reply:
See the ⛔ here.